### DISCUSSION TITLE
fulltextsearch support

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#452] [Ideas] [answered]
Should be added to the optional section

**Comment by TheNomad11** [comment]
I am trying to set up elasticsearch and - sofar - cannot make it work with the internal elasticsearch instance, while it works with an external one. Are there some specific settings or permission issues I should be aware of?

```
.Testing your current setup:  
Creating mocked content provider. ok  
Testing mocked provider: get indexable documents. (2 items) ok  
Loading search platform. (Elasticsearch) ok  
Testing search platform. fail 
In StaticNoPingConnectionPool.php line 64:
No alive nodes found in your cluster   
```

**Comment by szaimen** [comment]
Hello, please report that in a new issue. Thanks! :)

**Comment by szaimen** [comment]
This passes all [requirements for integrating this new container into AIO](https://github.com/nextcloud/all-in-one#requirements-for-integrating-new-containers) so it will be added at some point.

**Comment by szaimen** [comment]
See discussion in https://github.com/nextcloud/all-in-one/issues/1036

**Comment by szaimen** [comment, marked as answer]
This is now released with v2.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
